site/viamguides/src/home-assistant/home-assistant.md

```diff
@@ -0,0 +1,190 @@
+   ![time pattern settings](assets/ha-automation-trigger-config.png)
+1. Click "+ Add Action" to open the selection modal, then search for "Detect object" and select the top result.
+   ![detect objects actions](assets/ha-automation-action-search-detect.png)
+1. In the "Viam machine" field, select the configured machine from earlier. Enter the name of the vision service, "vision-1", in the "Detector Name" field. Check the "Camera Entity" box and select the connected camera you would like to use for this automation. Then set the "Response variable" to "people_detections".
+   ![detect objects configuration](assets/ha-automation-action-config-detect.png)
+1. Click "+ Add Action"  to open the selection modal, then search for "Condition" and select the top result.
+   ![condition action](assets/ha-automation-action-search-condition.png)
+1. Select "Template" for the "Condition type" and enter the following into the "Value template" field:
+   ```yaml
+   {{ people_detections.detections|count > 0 }}
+    ```
+   ![condition config](assets/ha-automation-action-config-condition.png)
+1. Click "+ Add Action"  to open the selection modal, then search for "Persist" and select the top result.
+   ![notification action](assets/ha-automation-action-search-persist.png)
+1. Click the action menu (three vertical dots) on the new "Persistent Notifications: 'Create'" card, and select "Edit in YAML" to display the YAML editor for creating templated notifications. Replace the `data: {}` line with the following code:
+   ```yaml
+   data:
+     title: People Alert
+     message: >
+       Detected {{ people_detections.detections|join(", ", "name")|default("nothing",
+       true) }}: ![captured image]({{ people_detections.img_src }})
+    ```
+    This will create a notification with a list of detected people and the associated image.
+   ![notification config](assets/ha-automation-action-config-persist.png)
+1. Click the "Save" button in the bottom-right corner to enter a name for the automation and complete the setup.
+
+   > aside negative
+   > **Troubleshooting:**
+   >
+   > - Check the **Traces** page of the automation to display relevant errors.
+   > - Check the "Settings" -> "System" -> "Logs" page to see more details about any errors.
+
+<!-- ------------------------ -->
+## Next Steps
+Duration: 3
+
+Congratulations, you now have automated people detection for your smart home! Building on this experience you can do the following to make it even smarter:
+
+- [automatically collect images](https://github.com/HipsterBrown/viam-home-assistant-integration?tab=readme-ov-file#viamcapture_image) or [sensor data](https://github.com/HipsterBrown/viam-home-assistant-integration?tab=readme-ov-file#viamcapture_data) from your Home Assistant devices in Viam
```

Comment:
I am definitely going to try piping in my air quality sensor data into HA, and try to visualize it too. Good suggestion.